pkg/artifactory/repositories.go

```diff
@@ -719,8 +754,20 @@ func unpackBaseRemoteRepo(s *schema.ResourceData, packageType string) RemoteRepo
 	if v, ok := d.GetOk("content_synchronisation"); ok {
 		contentSynchronisationConfig := v.([]interface{})[0].(map[string]interface{})
 		enabled := contentSynchronisationConfig["enabled"].(bool)
+		statisticsEnabled := contentSynchronisationConfig["statistics_enabled"].(bool)
+		propertiesEnabled := contentSynchronisationConfig["properties_enabled"].(bool)
+		sourceOriginAbsenceDetection := contentSynchronisationConfig["source_origin_absence_detection"].(bool)
 		repo.ContentSynchronisation = &ContentSynchronisation{
 			Enabled: enabled,
+			Statistics: ContentSynchronisationStatistics{
```

Comment:
So, I was meaning to ask: `ContentSynchronization` is a pointer here and I don't see the need. I won't ask that you change it, but when you come upon code like this you're aware and try to actually eliminate pointers